libraries/rush-lib/src/cli/actions/BridgePackageAction.ts

```diff
@@ -1,36 +1,40 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+import type { CommandLineStringParameter } from '@rushstack/ts-command-line';
+
 import type { RushCommandLineParser } from '../RushCommandLineParser';
 import type { RushConfigurationProject } from '../../api/RushConfigurationProject';
 import { BaseSymlinkPackageAction } from './BaseSymlinkPackageAction';
 import type { RushConnect } from '../../utilities/RushConnect';
-import type { CommandLineStringParameter } from '@rushstack/ts-command-line';
+import { BRIDGE_PACKAGE_ACTION_NAME, LINK_PACKAGE_ACTION_NAME } from '../../utilities/actionNameConstants';
 
 export class BridgePackageAction extends BaseSymlinkPackageAction {
   private readonly _version: CommandLineStringParameter;
 
   public constructor(parser: RushCommandLineParser) {
     super({
-      actionName: 'bridge-package',
-      summary: '(EXPERIMENTAL) Simulate installation of a locally built project, affecting many projects.',
+      actionName: BRIDGE_PACKAGE_ACTION_NAME,
+      summary:
+        '(EXPERIMENTAL) Simulate replacement of an installed external dependency with a package that lives in ' +
+        'a folder outside of this repo, affecting many projects.',
```

Comment:
```suggestion
        '(EXPERIMENTAL) Simulate installation of a local project folder as a dependency of one or more projects.',
```

We are simulating _**installation**_ not simulating _**replacement**_.  Replacement is actually occurring.

The main idea:

1. Somewhere **locally** on your computer you have a project folder, which you **built**.  This is what we will simulate installing (whereas a non-simulated installation would come from the NPM registry.)
2. One or more target projects will have this project folder symlinked in their node_modules folder.  These symlinkes "simulate" an installation, whereas a non-simulated installation would be reflected in the lockfile and would respect SemVer.